nix_update/__init__.py

```diff
@@ -125,7 +125,7 @@ def nix_shell(options: Options) -> None:
             check=False,
         )
     else:
-        expr = f"let pkgs = import {options.import_path} {{}}; in pkgs.mkShell {{ buildInputs = [ pkgs.{options.escaped_attribute} ]; }}"
+        expr = f'let pkgs = import "{options.import_path}" {{}}; in pkgs.mkShell {{ buildInputs = [ pkgs.{options.escaped_attribute} ]; }}'
```

Comment:
Should we also use json.dumps on import_path to escape characters like `"`?

Reply:
yeah that's probably better